test/Sema/access-level-import-diag-priority.swift

```diff
@@ -62,6 +69,26 @@ public func publicFuncUsesPrivateScambled(_ a: PublicImportType, d: FileprivateI
     var _: PrivateImportType
 }
 
+//--- PackageTypeImportedAsPackageClient.swift
+/// Report errors about using package decls in public but don't note the import
+/// as it doesn't affect the access level of the decls.
+
+public import PublicLib
+package import PackageLib
+
+public func publicFuncUsesPackageLevelPublicImportedType(_ a: PackageLevelPublicImportedType) {} // expected-error {{function cannot be declared public because its parameter uses a package type}}
```

Comment:
It should give the same error if `PublicImportType` is used here right?

Reply:
`PublicImportType` would be ok as long as it's publicly imported. It would show the same error for ` PackageImportType`, with the note.